e2e/tests/heavy_writing_and_recurring_jobs.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/volume.resource
+Resource    ../keywords/node.resource
+Resource    ../keywords/common.resource
+Resource    ../keywords/recurring_job.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+
+*** Test Cases ***
+Volume Attached Node Reboot While Heavy Writing And Recurring Jobs Exist
+    Create volume 0 with size 2 GB and 1 replicas
+    Create volume 1 with size 2 GB and 3 replicas
+    Keep writing data to volume 0
+    Keep Writing data to volume 1
+    Create snapshot and backup recurring job for volume 0
+    Create snapshot and backup recurring job for volume 1
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        Reboot volume 0 attached node
+        Check recurring jobs for volume 0 work
+        Check recurring jobs for volume 1 work
```

Comment:
What things will be verified?

Reply:
Check if there will be a backup/snapshot created by the corresponding recurring job in expected timing.